tests/clock/ClockUtils.py

```diff
@@ -0,0 +1,384 @@
+import logging
+import random
+import datetime
+import allure
+import time
+
+from tests.common.helpers.assertions import pytest_assert
+from tests.clock.ClockConsts import ClockConsts
+from tests.common.errors import RunAnsibleModuleFail
+
+
+class ClockUtils:
+    @staticmethod
+    def run_cmd(duthosts, cmd, param=''):
+        """
+        @summary:
+            Run a given command and return it's output.
+            * a successful command returns empty output (''), while failure returns error message
+        @return: commands output (str)
+        """
+        with allure.step('Run command: "{}" with param "{}"'.format(cmd, param)):
+            logging.info('Run command: "{}" with param "{}"'.format(cmd, param))
+            DUT_HOSTNAME = duthosts[0].hostname
+
+            cmd_to_run = cmd if param == '' else cmd + ' ' + param
+            logging.info('Actual command to run: "{}"'.format(cmd_to_run))
+
+            try:
+                cmd_output = duthosts.command(cmd_to_run)[DUT_HOSTNAME][ClockConsts.STDOUT]
+            except RunAnsibleModuleFail as cmd_err:
+                output = cmd_err.results[ClockConsts.STDOUT]
+                err = cmd_err.results[ClockConsts.STDERR]
+                cmd_output = output if output else err
+                logging.info('Command Error!\nError message: "{}"'.format(cmd_output))
+            logging.info('Output type: {}'.format(type(cmd_output)))
+            logging.info('Output: {}'.format(cmd_output))
+
+        with allure.step('Convert output to string'):
+            logging.info('Convert output to string')
+            cmd_output = str(cmd_output)
+            logging.info('Output type: {}'.format(type(cmd_output)))
+            logging.info('Output: {}'.format(cmd_output))
+        return cmd_output
+
+    @staticmethod
+    def parse_show_clock_output(show_clock_output):
```

Comment:
have you considered using the time library: https://docs.python.org/3/library/time.html
It would always be a preferable option to use an existing library.

Reply:
i used datetime library instead (in the last commit)